doc/installation.md

```diff
@@ -239,9 +239,14 @@ e.g.
     ```sh
     cd <espnet-root>/tools
     . activate_python.sh
-    . ./setup_cuda_env.sh <cuda-root>  # e.g. <cuda-root> = /usr/local/cuda
     ./installers/install_warp-transducer.sh
     ```
+- To install Warp RNNT
+    ```sh
+    cd <espnet-root>/tools
```

Comment:
I think this is not necessary. RNNT is not worth to dare to write

Reply:
OK I'll remove it. However I'll revisit warp-transducer vs warp-rnnt, it seems I have some (small) differences compared to my initial benchmark!

Reply:
These tools are shown for examples. We don't need to show all.